### DISCUSSION TITLE
Go back to previous position with one key stroke

### DISCUSSION THREAD

**Discussion by LintaoAmons** [folke/flash.nvim#59] [Ideas]
Like we can use 

- `gi` to go back to the previous insert position and continue insertion.
- `gv` to go back to the previous selected position.

I think it could be very useful if the plugin have some sort of smart go back function. 

After I `f/F` or `s` jump to some other place and finish something, mostly copying some stuff, then I can just press one key to back the position where I am before jumping.

Like `s`, I think this should also be able to across multiple splits~

**Comment by IndianBoy42** [comment]
The previous location is added to the jumplist so `<C-o>` should work. Also `require'flash'.remote()` is very useful here. you can do `yr<pattern>iw` for example to copy a word somewhere else and have you cursor back to where it was before `y`

**Reply by folke** [reply]
This is the correct answer.
